bestPractices.js

```diff
@@ -0,0 +1,333 @@
+const loadElements = () => {
+  let taskArray = [];
+  taskArray = retrieveData();
+  taskArray
+    .sort((x, y) => x.index - y.index)
```

Comment:
- using x and y might be confusing, would be great if you rename these variables like **taskA** and **taskB**,  That would make your code Easier to read! and would comply with JavaScript Best Practices, using meaningful Names.